_data/server_permissions.yaml

```diff
@@ -0,0 +1,151 @@
+# general permissions
+- permission: adminMessageReceive
+  description: Receives messages sent to the admin channel
+- permission: adminMessageSend
+  description: Allows sending messages to the admin channel
+- permission: hideAdmin
+  description: Doesn't show `@`
+- permission: info
+  description:
+- permission: privateMessage
+  description: Allows sending private messages with `.`
+- permission: rejoin
+  description: Allows rejoining instantly, regardless of `_rejoinTime`
+- permission: removePerms
+  description: Allows removing permissions from players
+- permission: requireIdentify
+  description: Requires callsign to identify before allowed to spawn
+- permission: setPerms
+  description: Alls granting permissions to players
+- permission: shortBan
+  description: Allows bans with limited duration only
+- permission: showAdmin
+  description: Gives player an `@` regardless of having the `shortban` or `ban` permission; the `hideAdmin` permission supersedes the `showAdmin` permission
```

Comment:
gives any player, or just admin players? Does this permission grant admin rights, or just the decoration?

Reply:
This permission is just for decoration